source/isNotEmpty.js

```diff
@@ -0,0 +1,28 @@
+import _curry1 from './internal/_curry1.js';
+import isEmpty from './isEmpty.js';
+
+
+/**
+ * Returns `true` if the given value is its type's empty value; `false`
```

Comment:
And the end of this sentence should be switched? "Returns `true` if the given value is not empty; `false` otherwise"